### DISCUSSION TITLE
Any experts available to configure and teach my team how to set up postal?

### DISCUSSION THREAD

**Discussion by golfaggressive** [postalserver/postal#2490] [Help with using Postal] [answered]
Willing to pay someone to configure a few postal instances, and then teach my technical team how to do the same. We are thinking of offering postal to email marketers. First, we would like to know what the heck we are doing. :)

Chris

**Comment by vibhoo** [comment, marked as answer]
@golfaggressive 
I can help you in setting up and running the postal, recently I have set up the postal and it's working fine for me, reach out to me on this [email](vibhoo.mishra04@gmail.com)
